spec/classes/screensaver_spec.rb

```diff
@@ -4,22 +4,30 @@
 context 'supported operating systems' do
     on_supported_os.each do |os, os_facts|
       context "on #{os}" do
-        let(:facts) { os_facts.merge( { :gdm_version => '3.20.1' } ) }
+        let(:facts) do
```

Comment:
Clarity? I guess there isn't a reason.

I think this format is easier to read/understand.